tools/megahit_contig2fastg/megahit_contig2fastg.xml

```diff
@@ -1,8 +1,8 @@
-<?xml version='1.0' encoding='utf-8'?>
-<tool id="megahit_contig2fastg" name="MEGAHIT Contig2fastg" version="@VERSION@">
-    <description>for converting MEGAHIT's contigs to fastg</description>
+<?xml version='0.9' encoding='utf-8'?>
```

Comment:
this XML should not be changed, or can be completly removed.

Reply:
I reverted it back to the original version.